app/models/annotation/handler/AnnotationInformationHandler.scala

```diff
@@ -28,8 +28,8 @@ trait AnnotationInformationHandler {
     Future.successful(t.id)
   }
 
-  def withAnnotation[A](identifier: String)(f: Annotation => A)(implicit request: AuthenticatedRequest[_]): Fox[A] = {
-    provideAnnotation(identifier, Some(request.user)).map(f)
+  def withAnnotation[A](identifier: String)(f: Annotation => A)(implicit request: SecuredRequest[_]): Fox[A] = {
```

Comment:
withAnnotation method is never used, i think, and shouldn’t be. should have been removed already in brave new world. my bad, but maybe you can do this here now

Reply:
I will remove it